spec/wasCrawlPreassembly/lib/metadata_extractor_service_spec.rb

```diff
@@ -8,60 +8,6 @@
     @crawl_id = 'test_crawl'
   end
 
-  context Dor::WASCrawl::MetadataExtractor, '.call_java_library with druid that has one warc file' do
-    it 'should create a valid output file' do
-      druid_id = 'druid:ab123ab1234'
-      metadata_extractor_service = Dor::WASCrawl::MetadataExtractor.new(@collection_id, @crawl_id, @staging_path.to_s, druid_id)
-
-      metadata_extractor_service.run_metadata_extractor_jar
-
-      output_file = 'tmp/' + druid_id + '.xml'
-      expect(File.exist?(output_file)).to be_truthy
```

Comment:
hrmm.  This test could be morphed to check if the java command to be executed is correct?    That would be a more appropriate test anyway -- the correctness of the output xml should be tested in the java project, no?